man/zypper-docker-pchk.1.md

```diff
@@ -0,0 +1,25 @@
+% ZYPPER-DOCKER(1) zypper-docker User manuals
+% SUSE LLC.
+% SEPTEMBER 2015
+# NAME
+zypper\-docker pchk \- Check for patches.
+
+zypper\-docker patch-check-container \- Check for patches available for the
+given container.
+
+# SYNOPSIS
+**zypper-docker patch-check** IMAGE
+
+**zypper-docker patch-check-container** CONTAINER
+
+# DESCRIPTION
+The **patch-check** command checks for patches that are available for the
+given openSUSE/SLES image. The provided image follows the same naming
+conventions as in Docker. To fetch which images are based on openSUSE or SLES,
+use the **images** command.
+
+The **patch-check-container** takes the container ID and lists the patches for
+the image in which the given container is based on.
+
```

Comment:
We have to add the details about the exit codes. This can be copied by zypper's man page.